spec/factories/submissions.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :submission, class: CCMS::Submission do
+    legal_aid_application { create :legal_aid_application }
+
+    trait :initialised do
+      aasm_state { 'initialised' }
+    end
```

Comment:
As default, won't the state by definition be set at `initialised`? If so - this is redundant.